android/app/src/main/java/app/organicmaps/util/UiUtils.java

```diff
@@ -377,4 +377,34 @@ public static void startActivityForResult(@NonNull Fragment fragment, @NonNull I
 
   // utility class
   private UiUtils() {}
+
+  @Override
+    void bind(@NonNull SearchResult result, int order) {
+        super.bind(result, order);
```

Comment:
Hm, but UiUtils is the utility class, not inherited from anything, and has only public static functions.
Am I missing something? Where does override bind come from?